meetings/2017-09-27.md

```diff
@@ -0,0 +1,147 @@
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1oMS1G-Tpgx8Ha8FqCb1XCLXSu5yZ6AjO1HmB1clUPaU>_
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Evan Lucas @evanlucas (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* MichaëZasso @targos (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+Extracted from **tsc-agenda** labelled issues and pull requests from
+the **nodejs org** prior to the meeting.
+
+### Workgroup Update
+
+#### Release Work Group
+
+### nodejs/node
+* path: deprecate internal _makeLong, replace [#14956](https://github.com/nodejs/node/pull/14956)
+
+### nodejs/TSC
+
+### nodejs/admin
+* doc: doc expectations on TSC and CommComm members [#12](https://github.com/nodejs/admin/pull/12)
+
+### Initial discussion on 'values'
+* discuss, Myles may have a draft to seed the discussion.
+
+### Long term TSC goals
+* kick off discussion.
+
+## Minutes
+
+### Announcements
+No announcements this week
+
+### Workgroup Update
+
+#### Release Work Group
+* I must have miscommunicated (or just not provided dial-in info) so we
+  did not have Gibson to present
+
+### nodejs/node
+* path: deprecate internal _makeLong, replace [#14956](https://github.com/nodejs/node/pull/14956)
+  * Matteo, new module work to enable ES7 there are certain
+    Functions from core that are pre-fixed that are required
+    Proposes to remove underscore so that it is part of the
+    public API.  Likely to be used by lodash so would be heavy
+    Usage.
+  * Anna what need to see if anybody objects to it happening ?
+  * Put into 9 and revert if pushback ?
+  * Rich: Seems like no main objections in the issue so it could land ?
+  * Ben has asked that we answer these 2 questions:
+    * Are we comfortable promoting this to officially supported API?
+      That effectively freezes it for all eternity, or as good as.
+      path._makeLong() has been tweaked over the years. A likely outcome
+      is that we end up with a tweaked internal copy sooner or later and then we
+      are back to square one.
+    * Is it so widely useful that it should be part of core?1 Why can't
+      it be an npm package?  That's our usual answer when people request
+      non-necessary features; why is this different?
+
+  * After discussion we added this to the issue:
+    * Based on discussion in the TSC meeting consensus is that we can
+      move forward. We can land once comments are addressed unless
+      there are objections in next 48 hours.
+
+### nodejs/TSC
+
+### nodejs/node 
+* [#15520](https://github.com/nodejs/node/pull/15520)
+  * May be issue/bug on centos 5
+  * Rich, some discussion of dropping centos 5 so maybe we
```

Comment:
This looks cut-off. Maybe delete `so maybe we`?

Reply:
Done